src/mockClient.ts

```diff
@@ -11,14 +11,15 @@ export type RequestHandlerResponse<T> = { data: T, errors?: any[] };
 export type MockApolloClient = ApolloClient<NormalizedCacheObject> &
   { setRequestHandler: (query: DocumentNode, handler: RequestHandler) => void };
 
-export const createMockClient = (): MockApolloClient => {
+export const createMockClient = (options?: Partial<ApolloClientOptions<NormalizedCacheObject>>): MockApolloClient => {
   const mockLink = new MockLink();
 
   const client = new ApolloClient({
     cache: new Cache({
       addTypename: false, // TODO: Handle addTypename?
     }),
     link: mockLink,
+    ...options
```

Comment:
I think link should always be taking over one passed in options, 
so could we perhaps swap the order around?

```suggestion
    ...options,
    link: mockLink
```